Gruntfile.js

```diff
@@ -58,8 +59,31 @@ module.exports = function(grunt)
           }
         ]
       }
+    },
+    prettier: {
+      options: {
+        progress: true,
+        singleQuote: true
+      },
+      tests: {
+        src: [
+          'test/**/*.js'
+        ]
+      },
+      // src: {
+      //   src: [
+      //     'src/**/*.js',
+      //     '!src/libs/*.js'
+      //   ]
+      // },
+      // app: {
+      //   src: [
+      //     'app/**/*.js',
+      //     '!app/public/**'
+      //   ]
+      // }
```

Comment:
Do we mean for this commented out code to still be here?

Reply:
Yes. @andrewstart [caught this earlier](https://github.com/SpringRoll/SpringRollConnect/pull/66#discussion_r258167100) :)